README.md

```diff
@@ -1,2 +1,22 @@
-# serializer-jms-adapter
-An adapter to make liip/serializer a drop-in replacement for jms/serializer
+# Liip Serializer JMS Adapter
+
+An adapter to make liip/serializer a drop-in replacement for jms/serializer.
+
+This library provides a serializer that attempts to use the `LiipSerializer` and falls back to JMS
+serializer on error. You need to use the `Liip\\Serializer\\Adapter\\JMS\\AdapterSerializationContext`
+provided in this library instead of the regular `JMS\Serializer\SerializationContext`. You can use
+the `Serializer\\Adapter\\JMS\\AdapterSerializationContextFactory` when working with libraries that
+create the context themselves.
+
+# Usage
+
+...
```

Comment:
We certainly need something here so it's somewhat useful :) Most minimal example would already be fine.

Reply:
added a code example. actually i think there is not much more to say about the adapter.